one/tests/SmartDonation.test.ts

```diff
@@ -0,0 +1,80 @@
+import { createPrivateKey, Hash256 } from '@neo-one/client';
+import BigNumber from 'bignumber.js';
+import { withContracts } from '../generated/test';
+
+jest.setTimeout(30000);
+
+test('SmartDonation - Test', async () => {
+  await withContracts(async ({ networkName, one, client, developerClient, smartDonation, masterAccountID }) => {
+    // setup our test account to have some ONE, also create a second wallet for testing
+    await developerClient.fastForwardOffset(60 * 60);
+    const testMessage = 'take my money.';
+
+    const receiveWallet = await client.providers.memory.keystore.addAccount({
```

Comment:
Use the `accountIDs` property that's passed into the callback rather than adding a new wallet.